cmd/server/main.go

```diff
@@ -37,6 +37,7 @@ var (
 	keyVersion    = flag.String("key-version", "", "Azure Key Vault KMS key version")
 	managedHSM    = flag.Bool("managed-hsm", false, "Azure Key Vault Managed HSM. Refer to https://docs.microsoft.com/en-us/azure/key-vault/managed-hsm/overview for more details.")
 	logFormatJSON = flag.Bool("log-format-json", false, "set log formatter to json")
+	logLevel      = flag.Int("v", 0, "In order of increasing verbosity: 0=warning, 2=info, 4=debug, 6=trace, 100=all")
```

Comment:
Internally `mlog` only checks for `8` even though it sets it much higher, some Kube code uses `10` IIRC:

```suggestion
	logLevel      = flag.Int("v", 0, "In order of increasing verbosity: 0=warning/error, 2=info, 4=debug, 6=trace, 10=all")
```